executor/extension/validator/utils.go

```diff
@@ -209,3 +212,60 @@ func doSubsetValidation(alloc txcontext.WorldState, db state.VmStateDB, updateOn
 	}
 	return nil
 }
+
+// updateEthereumDb is used to fix exceptions in ethereum dataset inconsistencies
+func updateEthereumDb(s executor.State[txcontext.TxContext], ctx *executor.Context, isPreTransaction bool) error {
```

Comment:
I think the name is misleading. Afaik we are not updating `EthereumDb` right?

Reply:
I thought about `fixEthereumExceptions`. Im terrible at naming variables tho, maybe @wsodsong has better idea